### ISSUE TITLE
Wrong docstring for position_ids

### ISSUE THREAD

**Issue by RmZeta2718** [huggingface/transformers#44373]
### System Info

latest commit, see link below

### Who can help?

@stevhliu 

### Information

- [ ] The official example scripts
- [ ] My own modified scripts

### Tasks

- [ ] An officially supported task in the `examples` folder (such as GLUE/SQuAD, ...)
- [ ] My own task or dataset (give details below)

### Reproduction

Wrong docstring:

https://github.com/huggingface/transformers/blob/11b1906d5c0dae39c13270e47cc02c4cde70e548/src/transformers/modeling_flash_attention_utils.py#L359-L360

https://github.com/huggingface/transformers/blob/11b1906d5c0dae39c13270e47cc02c4cde70e548/src/transformers/modeling_flash_attention_utils.py#L412-L413

The docstring is for attention_mask, not position_ids

https://github.com/huggingface/transformers/blob/11b1906d5c0dae39c13270e47cc02c4cde70e548/src/transformers/modeling_flash_attention_utils.py#L201

### Expected behavior

Expected docstring:

https://github.com/huggingface/transformers/blob/11b1906d5c0dae39c13270e47cc02c4cde70e548/src/transformers/models/layoutlmv2/modeling_layoutlmv2.py#L897-L901

or

https://github.com/huggingface/transformers/blob/11b1906d5c0dae39c13270e47cc02c4cde70e548/src/transformers/masking_utils.py#L715-L716

**Comment by ManasVardhan** [comment]
I'll take this - fixing the docstring now.

**Comment by mvanhorn** [comment]
I've submitted a fix for this in #44547. Corrected the position_ids docstring to accurately describe position indices instead of attention mask semantics.
